File: mpp-proiect-repository-saboalina-Lab3C-/Laborator2C/domain/Zbor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Lab2C.domain
{
    class Zbor:Entity<int>
    {
        public String destinatie { get; set; } 
        public String dataSiOraPlecarii { get; set; }
        public String aeroport { get; set; }
        public int nrLocuriDisponibile { get; set; }

        public override string ToString()
        {
            return destinatie+" "+dataSiOraPlecarii+" "+aeroport+" "+nrLocuriDisponibile;
        }
    }
}

File: mpp-proiect-repository-saboalina-Lab3Java/src/main/java/domain/Zbor.java
package domain;

import java.time.LocalDateTime;

public class Zbor extends Entity<Integer>{
    private String destinatie;
    private LocalDateTime dataSiOraPlecarii;
    private String aeroport;
    private Integer nrLocuriDisponibile;

    public Zbor(String destinatie, LocalDateTime dataSiOraPlecarii, String aeroport, Integer nrLocuriDisponibile) {
        this.destinatie = destinatie;
        this.dataSiOraPlecarii = dataSiOraPlecarii;
        this.aeroport = aeroport;
        this.nrLocuriDisponibile = nrLocuriDisponibile;
    }

    public String getDestinatie() {
        return destinatie;
    }

    public LocalDateTime getDataSiOraPlecarii() {
        return dataSiOraPlecarii;
    }

    public String getAeroport() {
        return aeroport;
    }

    public Integer getNrLocuriDisponibile() {
        return nrLocuriDisponibile;
    }
}
